src/accelerate/utils/imports.py

```diff
@@ -177,7 +177,10 @@ def is_pippy_available():
     package_exists = _is_package_available("pippy", "torchpippy")
     if package_exists:
         pippy_version = version.parse(importlib.metadata.version("torchpippy"))
-        return compare_versions(pippy_version, ">", "0.1.1")
+        if compare_versions(pippy_version, ">", "0.1.1") and compare_versions(pippy_version, "<=", "0.2.0"):
+            return "0.1.1-0.2.0"
+        elif compare_versions(pippy_version, ">", "0.2.0"):
+            return ">0.2.0"
```

Comment:
Rather than this, let's keep the current behavior of `true` or `false`. 

Let's do a different argument of `max_version` to be passed in the `is_pippy_available` function with a default of `None`

Reply:
Ah I see. I didn't fully understand what you meant before but I think I get it now.
Let me know!